src/view/popup/pages/RecordingPage.tsx

```diff
@@ -31,15 +31,16 @@ export const RecordingPage = ({
             <div className="h-24 text-base bg-nyanza flex flex-column flex-wrap content-evenly justify-center text-center rounded-2xl shadow font-medium">
                 {formatBytes(bytesTransferred)}
                 <br />
-                {formatEmissions(emissions)}
+                {formatEmissions(emissions) + " grams of CO2"}
```

Comment:
I'd recommend putting this in a `<p>` tag with string template literals for a cleaner way to write text that uses a variable inside the string: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals